doc/SYNDICATION.md

```diff
@@ -0,0 +1,147 @@
+- **Quick Overview**
+
+    Syndication (also known as Republishing), is the term for allowing other publishers to use our content in their own publications. At the time of writing this there are over 200 Syndication licences and nearly 600 end users for those licences. In total those ~200 licences amount to over £3.2m.
+
```

Comment:
amount to over £3.2m per... year? quarter? month?

Reply:
Removed those figures as they should now be outdated.